geminimd/commands.py

```
@@ -0,0 +1,401 @@
+                continue
+            fname = sanitize_filename(meta_att.get("name", att_id))
+            local_path = attachments_dir / fname
+            if not dry_run:
+                if force or not local_path.exists():
+                    ok = drive.download_attachment(att_id, local_path)
+                    if not ok:
+                        continue
+                drive.touch_mtime(local_path, meta_att.get("modifiedTime"))
+            try:
+                rel = local_path.relative_to(out_dir)
+            except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)